### PR TITLE
Fix calendar file download button alignment issue

### LINKED ISSUE #333: Event info panel issue
## Describe the bug

"Åpent for" is not on a new line after "Last new i kalender" #258

## To Reproduce

1. Go to '/events'
2. Click on an event attendable only for certain grade levels
3. Look at info panel on the right

## Expected behavior

"Åpent for" to be on a new line, as is done with "Åpner"

## Screenshots

Skjermbilde 2021-11-24 kl  19 15 10

## Desktop (please complete the following information):

- OS: MacOS
- Browser Chrome
- Version 96

### PR DESCRIPTION
**Pull request by MorganFH**
## Proposed Changes

- Fix alignment of calendar file download button by ~~wrapping it in a `Box`~~ introducing `Grid`structure

Before  After -> 

- Add `Grid` structure
Before   After ->

## Types of Changes

- [ ] New feature
- [x] Bug fix
- [ ] Enhancement/optimization
- [ ] Refactor
- [ ] Style
- [ ] Build/dependencies
- [ ] Other

## Issue(s) fixed or closed by this PR

- closes #333 

## Checklist

- [ ] I have added tests to cover my changes (for features/bugs)
- [x] I am pleased with the readability of the code (if not, state where you need input)
- [x] I have tested the changes and verified that they work and do not break anything (to the best of my ability)

### CODE REVIEW

File: frontend/src/components/pages/events/EventDetails.tsx [hunk 1]
```diff
@@ -359,13 +359,15 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
...
                   </>
                 )}
-                <Button
-                  variant="text"
-                  href={calendarFile(event.title, event.startTime, event.endTime, event.location, event.description)}
-                  download="event.ics"
-                >
-                  Last ned i kalender
-                </Button>
+                <Box>
```
Comment (larwaa) [inline frontend/src/components/pages/events/EventDetails.tsx:362]: Should arguably refactor this and place all items here in a Grid instead, however, might be beyond the scope of this PR

**Review by larwaa — CHANGES_REQUESTED** [review]
Would just like your thoughts on this 👇

File: frontend/src/components/pages/events/EventDetails.tsx [hunk 2]
```diff
@@ -317,67 +317,92 @@ const EventDetails: React.FC<Props> = ({ eventId }) => {
...
-                  <EventIcon fontSize="small" />
-                  {dayjs(event.startTime).format("DD.MMM YYYY, kl. HH:mm")}
-                </Typography>
-                {event.endTime && (
-                  <>
-                    <Typography variant="overline">Slutter</Typography>
-                    <Typography variant="body1" className={classes.wrapIcon}>
-                      <EventIcon fontSize="small" /> {dayjs(event.endTime).format("DD.MMM YYYY, kl. HH:mm")}
+                <Grid container spacing={1}>
+                  <Grid item>
```
Comment (MorganFH) [inline frontend/src/components/pages/events/EventDetails.tsx:321]:
```suggestion
                  <Grid item xs={12}>
```

**Review by larwaa — APPROVED** [review]
Haven't checked the styling myself, but code looks good

**Comment by MorganFH** [comment]
> Haven't checked the styling myself, but code looks good

Judging from my screenshots the vertical spacing increases slightly. Fine for now IMO.

**Pull request merged**